be/src/exec/pipeline/aggregate/aggregate_blocking_sink_operator.h

```diff
@@ -58,22 +58,28 @@ class AggregateBlockingSinkOperator : public Operator {
     std::atomic_bool _is_finished = false;
     // whether enable aggregate group by limit optimize
     bool _agg_group_by_with_limit = false;
+    std::atomic<int64_t>* _runtime_limit = nullptr;
```

Comment:
suggest to use std::atomic<int64_t>& type, then initialize it in ctor.
distinct_blocking_sink_operator also needs this optimization.

Reply:
Yes, correct, will do it.